packages/webapp/src/components/Finances/Transaction/Mobile/styles.module.scss

```diff
@@ -42,7 +43,7 @@
     border-radius: 1px;
   }
 
-  &:hover {
+  &:hover:not(.expanded) {
```

Comment:
I kinda liked the hover effect on the expanded item because it showcases more clearly that the item can be clicked on to collapse, but not sure if Figma calls for anything specific in this situation

Reply:
[Loïc's input](https://lite-farm.atlassian.net/browse/LF-3745?focusedCommentId=16353):
_It should not change colour on hover when the row is expanded, the reason is that it can be distracting when you process the info inside the expanded row._